custom_components/sat/climate.py

```
@@ -806,7 +777,7 @@ async def _async_control_relative_modulation(self) -> None:
             return
 
         # Update relative modulation state
-        await self._relative_modulation.update(self.warming_up, self.pwm.state)
+        await self._relative_modulation.update(self.max_error > DEADBAND, self.pwm.state)
```

Ruff (0.8.2)

`DEADBAND` may be undefined, or defined from star imports

(F405)